### DISCUSSION TITLE
¿Cuándo recibiré mis beneficios después de haber sido aprobado en el paquete de desarrollador para estudiantes?

### DISCUSSION THREAD

**Discussion by leafar-rz** [community/community#118353] [GitHub Education]
### Select Topic Area

Question

### Body

Me registre en el paquete de desarrollador para estudiantes y fui aprobado pero no me ha llegado el correo para obtener el beneficios, cuanto debo esperar?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @leafar-n!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the GitHub Education category. I've gone ahead and moved it for you. Good luck!
